common/test/acceptance/tests/studio/test_studio_outline.py

```diff
@@ -272,24 +337,87 @@ def test_unreleased_unpublished_changes_unlocked(self):
         release in the future'
         """
         self._verify_unit_warning(
-            self.UnitState(is_released=False, publish_state=self.PublishState.UNPUBLISHED_CHANGES, is_locked=False),
+            self.UnitState(
+                is_released=False,
+                publish_state=self.PublishState.UNPUBLISHED_CHANGES,
+                is_locked=False,
+                is_restricted=False
+            ),
             self.FUTURE_UNPUBLISHED_WARNING
         )
 
     def test_unreleased_published_locked(self):
         """ Tests that unreleased published locked units display staff only warnings """
         self._verify_unit_warning(
-            self.UnitState(is_released=False, publish_state=self.PublishState.PUBLISHED, is_locked=True),
+            self.UnitState(
+                is_released=False,
+                publish_state=self.PublishState.PUBLISHED,
+                is_locked=True,
+                is_restricted=False
+            ),
             self.STAFF_ONLY_WARNING
         )
 
     def test_unreleased_published_unlocked(self):
         """ Tests that unreleased published unlocked units display no warnings """
         self._verify_unit_warning(
-            self.UnitState(is_released=False, publish_state=self.PublishState.PUBLISHED, is_locked=False),
+            self.UnitState(
+                is_released=False,
+                publish_state=self.PublishState.PUBLISHED,
+                is_locked=False,
+                is_restricted=False
+            ),
             None
         )
 
+    def test_unreleased_unpublished_unlocked_restricted(self):
```

Comment:
Nit: you could use ddt to collapse these tests. Probably not worth it in this case because this test file is already following the pattern you are using, but something to look into for the future. Search for "ddt" inside the acceptance test folder to see examples.